package.json

```diff
@@ -84,7 +84,8 @@
     "style-loader": "^0.12.3",
     "url-loader": "^0.5.7",
     "webpack": "^1.8.2",
-    "react-addons-perf" : "0.14.7"
```

Comment:
I did not remover it.  It may be an accident when I cleaned up the package.json file.